test/test_verify.py

```diff
@@ -33,6 +41,12 @@ def ts_response() -> TimeStampResponse:
     return decode_timestamp_response((_FIXTURE / "response.tsr").read_bytes())
 
 
+@pytest.fixture
+def ts_response_for_algo(request) -> TimeStampResponse:
+    algo = request.param
+    return decode_timestamp_response((_FIXTURE / "sigstage" / f"response-{algo}.tsr").read_bytes())
```

Comment:
Non-blocker: I would rename this to `ts_response_by_filename()`, and use the whole tsr filename as an argument.

Reply:
this makes the parametrization a bit more verbose but I'm ok with that if it looks better